pipelines/LSSTCam/cpDarkForDefects.yaml

```diff
@@ -0,0 +1,11 @@
+# Note that if you edit this file you may also need to edit
+# cpDark.yaml.
+description: LSSTCam dark calibration construction for defects.
+instrument: lsst.obs.lsst.LsstCam
+imports:
+  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpDarkForDefects.yaml
+tasks:
+  cpDarkForDefectsIsr:
+    class: lsst.ip.isr.IsrTask
+    config:
+      doCrosstalk: false
```

Comment:
`doLinearize: false` isn't needed?

Reply:
Hmm.  It isn't there for the regular `LSSTCam/cpDark.yaml`.  I'm wondering what is going on in the pipeline right now...